concepticondata/conceptlists/Bowern-2021-203c.tsv

```diff
@@ -0,0 +1,204 @@
+Bowern-2021-203c-27	27	bottle 	618	acculturation	Culture Vocabulary	1321	BOTTLE
+Bowern-2021-203c-28	28	bottom grinding stone 	375	manufacture	Culture Vocabulary	3023	BOTTOM GRINDSTONE
+Bowern-2021-203c-29	29	bow 	615	subsistence tool	Culture Vocabulary	994	BOW
+Bowern-2021-203c-30	30	bowl (coolamon) 	771	culture-material	Culture Vocabulary	1530	BOWL
+Bowern-2021-203c-31	31	broom 	714	culture-material	Culture Vocabulary	245	BROOM
+Bowern-2021-203c-32	32	canoe 	616	transport	Culture Vocabulary	1970	CANOE
+Bowern-2021-203c-33	33	carrying net 	617	subsistence tool	Culture Vocabulary	293	NETBAG
+Bowern-2021-203c-34	34	cat 	702	acculturation	Culture Vocabulary	1208	CAT
+Bowern-2021-203c-35	35	chicken 	632	acculturation	Culture Vocabulary	1318	CHICKEN
+Bowern-2021-203c-36	36	chief/leader 	715	culture-mythology	Culture Vocabulary	301	CHIEFTAIN
+Bowern-2021-203c-37	37	cicatrice 	772	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-38	38	clapper 	705	acculturation	Culture Vocabulary		
+Bowern-2021-203c-39	39	clapsticks 	773	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-40	40	clothing 	676	culture-mythology	Culture Vocabulary	1895	CLOTHES
+Bowern-2021-203c-41	41	club 	621	subsistence tool	Culture Vocabulary	1763	CLUB
+Bowern-2021-203c-42	42	club (nulla-nulla) 	774	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-43	43	coca 	153	flora-fauna	Culture Vocabulary	137	COCA
+Bowern-2021-203c-44	44	coca 	717	narcotics	Culture Vocabulary	137	COCA
+Bowern-2021-203c-45	45	cook food 	623	food	Culture Vocabulary	1100	COOK (SOMETHING)
+Bowern-2021-203c-46	46	cook in earth oven 	775	food	Culture Vocabulary		
+Bowern-2021-203c-47	47	corn 	700	food	Culture Vocabulary	605	GRAIN
+Bowern-2021-203c-48	48	corpse 	624	food	Culture Vocabulary		
+Bowern-2021-203c-49	49	cradleboard 	697	culture-material	Culture Vocabulary		
+Bowern-2021-203c-50	50	cremate 	689	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-51	51	Curupira (spirit type) 	718	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-52	52	damper 	776	food	Culture Vocabulary		
+Bowern-2021-203c-53	53	dance (generic) 	625	culture-mythology	Culture Vocabulary	3472	DANCE (MOVEMENT)
+Bowern-2021-203c-54	54	dart (blowgun) 	719	subsistence tool	Culture Vocabulary	2587	DARTS
+Bowern-2021-203c-55	55	Datura wrightii 	653	narcotics	Culture Vocabulary		
+Bowern-2021-203c-56	56	deity/powerful spirit/culture figure 	720	culture-mythology	Culture Vocabulary	3231	DEITY
+Bowern-2021-203c-57	57	digging stick 	627	subsistence tool	Culture Vocabulary	339	DIGGING STICK
+Bowern-2021-203c-58	58	dipper 	628	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-59	59	dipper (small coolamon) 	777	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-60	60	dream 	629	culture-mythology	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-61	61	dream	397	mental	Culture Vocabulary	1920	DREAM (SOMETHING)
```

Comment:
In this case, we could map to the noun DREAM.

Reply:
Spanish gloss here is only "soñar". But the one above this one, Bowern-2021-203c-60, should be DREAMING OR DREAM now that I'm checking the glosses again.